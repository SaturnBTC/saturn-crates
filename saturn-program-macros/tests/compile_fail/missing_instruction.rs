use saturn_program_macros::saturn_program;

#[saturn_program(bitcoin_transaction = true, btc_tx_cfg(max_inputs_to_sign = 1, max_modified_accounts = 1))]
mod handlers {}

fn main() {} 