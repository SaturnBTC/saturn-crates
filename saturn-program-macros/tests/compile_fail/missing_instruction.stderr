error: missing `instruction = "Path"` in attribute
 --> tests/compile_fail/missing_instruction.rs:3:1
  |
3 | #[saturn_program(bitcoin_transaction = true, btc_tx_cfg(max_inputs_to_sign = 1, max_modified_accounts = 1))]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `saturn_program` (in Nightly builds, run with -Z macro-backtrace for more info)
